### ISSUE TITLE
There is a single orange dot when app is loading

### ISSUE THREAD

**Issue by matejdro** [pebble-dev/mobile-app#127]
![image](https://user-images.githubusercontent.com/507922/111686052-e3fcd880-8828-11eb-977a-b529f0579d68.png)

We probably want to place some kind of splash image here?

**Comment by crc-32** [comment]
I believe this is a spinner that is for some reason not spinning, I recall it spinning for me before though which is odd

**Comment by AliceGrey** [comment]
> I believe this is a spinner that is for some reason not spinning, I recall it spinning for me before though which is odd

Yeah I think this is a regression. In older versions it was a working spinner.

**Comment by hellcp** [comment]
This apparently fixed itself in the latest version, worth closing?
